sdks/go/pkg/beam/core/runtime/graphx/serialize.go

```diff
@@ -145,20 +145,20 @@ func encodeCustomCoder(c *coder.CustomCoder) (*v1.CustomCoder, error) {
 func decodeCustomCoder(c *v1.CustomCoder) (*coder.CustomCoder, error) {
 	t, err := decodeType(c.Type)
 	if err != nil {
-		return nil, fmt.Errorf("decodeCustomCoder bad type: %v", err)
+		return nil, fmt.Errorf("failed to decode custom coder proto %v for type %v: %v", c, c.Type, err)
```

Comment:
Missed removing the "proto " bits throughout this file. I think it's because it's in the second file, understandable oversight. That's why we review.

Reply:
Done, removed mentions of "proto" in error messages.